### PR TITLE
Created ESPN Test Queries #819

### PR DESCRIPTION
**Pull request by Master-Keshav**
## Description

> The changes made in this pull request should be briefly described.

## Checklist

> Before submitting this pull request, kindly verify that the ensuing checkpoints have been reached.

- [X] Have you adhered to the repository's defined coding convention rules?
- [X] Have you updated the 'documentation.md' file with the method/function documentation?
- [X] Have you sent a message along with the result or response?
- [X] Have you used the try-catch technique?
- [X] Has the method/class been added to the documentation (md file)?

## Screenshots
image

> Uploading a screenshot illustrating the approach you developed for validation is required.

## Additional Notes/Comments
ESPN Test Queries Created
> Any additional remarks or suggestions concerning this pull request are welcome.

---

I certify that I have carried out the relevant checks and provided the requisite screenshot for validation by submitting this pull request.
I appreciate your contribution.

### CODE REVIEW

File: src/test/espn_test.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    | `.get_scoreboard()`  | Tests the get_scoreboard() method of the ESPN class  |
+    | `.get_tournaments()` | Tests the get_tournaments() method of the ESPN class |
+    | `.get_teams()`       | Tests the get_teams() method of the ESPN class       |
+    """
+
+    def test_get_scoreboard(self):
+        espn = ESPN()
+        date = "20230721"  # Replace with a valid date
+        scoreboard = espn.get_scoreboard(date)
+        self.assertIsInstance(scoreboard, list)
```
Comment (nikhil25803) [inline src/test/espn_test.py:18]: Not only check the response instance type, but also check whether it has all the keys or not. 
For example, in the `codechef_test.py` script, we have also checked the key value of the response.
```python
self.assertEqual(
            list(method_response.keys()),
            ["name", "username", "profile_image_link", "rating", "details"],
            "Codechef:get_profile - keys mismatch",
```

**Review by nikhil25803 — CHANGES_REQUESTED** [review]

**Review by nikhil25803 — CHANGES_REQUESTED** [review]
The test files are returning errors, please make sure to run the file you are submitting prior to raise an issue.

**Comment by Master-Keshav** [comment]
> The test files are returning errors, please make sure to run the file you are submitting prior to raise an issue.

In this complete file, I've added comments and documentation to clarify the purpose of each test case and improved error handling by specifying assertions based on the expected data structure. Make sure to replace the date variable with a valid date before running your tests.

**Comment by Master-Keshav** [comment]
If any other error comes please send me a complete error log. That would be a great help @nikhil25803  
Thanks!

**Pull request closed without merging**